### DISCUSSION TITLE
Dynamic Power Limiter/Solar Passthrough with mutiple inverters

### DISCUSSION THREAD

**Discussion by ruffy8472** [hoylabs/OpenDTU-OnBattery#137] [Ideas]
Hi there, first of all:
I am following this exciting project for some time now and I really look forward to use this in my own solar project someday. You really did a great job!

Is there any plan for extend the power limit/solar passthrough function for multiple inverter support? Right now its only possible to add one inverter/energy meter to that function right? My plan would be to bundle all solar cells of the multiple balcony solar systems of our house into one big solar chunk loading a battery which then feeds 4 HM-600. Those then feed into the sub-distributions of each household. That way the self consumption could be greatly improved and more solar cells could be added easily. I made a little sketch for better understanding:
![Unbenanntes Diagramm](https://user-images.githubusercontent.com/117917513/224566846-6189d9e7-d2af-49f7-aaad-356704b9a7a3.png)
So openDTU would get consumption data from 4 indipendent energy meters and needs to adjust the power for each corresponding inverter. I think my usecase is somehow exotic but I just wanted to ask if there are maybe plans for adding multi inverter support in the descriped way. If not I would have to control everything with iobroker. (a direct implementation in openDTU would be much nicer tho 😉)

Thank you!

**Comment by helgeerbe** [comment]
Hi @ruffy8472, nice idea but I think this is beyond the scope of this project.

I'm even not sure, if this is a good idea. Using a single balcony solar system myself, I saw that I couldn't consume energy for around 8 € in a half year.  So taking your your 4 households * 16 € per year => 64€.

Cost for a battery and charger around 1000€.  So it will take more than 15 years to get the invest back. I don't see a case for that.

You would need at least an additional balcony  solar system per household. 500€ (panels and charger) * 4  + 1000€ for a battery => 3000€. If you save 200€ per year and system you would save 800€ per year. 3000 / 800 => 4 year for the return of investment.

But systems at this size are no hobby projects anymore. I would go for a professional solution.  Just my 2 cent
